### PR TITLE
Delay on readout characterization

### PR DESCRIPTION
**Pull request by Jacfomg**
The fidelity and QND do not seem affected when neglecting the resonator delation time as we are not applying qubit pulses during the measurements. However, there is some anomalous shift on the iq plane that goes away if you wait the resonator depletion time.

imagen

imagen

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.
- [ ] Compatibility with Qibo modules (Please edit this section if the current pull request is not compatible with the following branches).
    - [ ] Qibo: `master`
    - [ ] Qibolab: `main`
    - [ ] Qibolab_platforms_qrc: `main`

### CODE REVIEW

File: src/qibocal/protocols/characterization/readout_characterization.py [hunk 1]
```diff
@@ -23,6 +24,9 @@
 class ReadoutCharacterizationParameters(Parameters):
     """ReadoutCharacterization runcard inputs."""
 
+    delay: Optional[float] = None
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/readout_characterization.py:27]:
```suggestion
    delay: float = 0
```

File: src/qibocal/protocols/characterization/readout_characterization.py [hunk 2]
```diff
@@ -55,6 +59,9 @@ class ReadoutCharacterizationData(Data):
 
     qubit_frequencies: dict[QubitId, float] = field(default_factory=dict)
     """Qubit frequencies."""
+
+    delay: Optional[float] = None
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/readout_characterization.py:63]:
```suggestion
    delay: float
```

File: tests/runcards/protocols.yml [hunk 3]
```diff
@@ -475,6 +475,7 @@ actions:
   - id: readout characterization
     operation: readout_characterization
     parameters:
+      delay: 1000
```
Comment (andrea-pasquale) [inline tests/runcards/protocols.yml:478]: Please introduce another entry here where `delay` is not provided, you see that the code fails without my suggestions...

**Review by andrea-pasquale — CHANGES_REQUESTED** [review]

**Review by andrea-pasquale — APPROVED** [review]

**Comment by igres26** [comment]
Could we make both plots square and display them side by side?

**Review by Edoardo-Pedicillo — APPROVED** [review]

**Comment by Jacfomg** [comment]
> > Could we make both plots square and display them side by side?
> 
> We have three plots now, I can plot the blobs alone on top and the two matrices together below that.

image

**Comment by Jacfomg** [comment]
> > Could we make both plots square and display them side by side?
> 
> We have three plots now, I can plot the blobs alone on top and the two matrices together below that.

image

**Pull request merged**